### PR TITLE
Changes to index scipt to push to S3, and correctly populate CSV

### PR DESCRIPTION
**Pull request by duncan9562**

### CODE REVIEW

**Review by rhysemmas — DISMISSED** [review]
LGTM

File: DQ_IL2_index_raw_msg.py [hunk 1]
```diff
@@ -241,20 +248,45 @@ def main(argv):
       ATTEMPT_NO+=1
 
     ##############################################################################################################################
-    # Move files to Local archive folder
+    # Move files to WFLS
```
Comment (david-wirelab) [inline DQ_IL2_index_raw_msg.py]: This should be "Moves files to local folder" or something similar

**Review by david-wirelab — CHANGES_REQUESTED** [review]

File: DQ_IL2_index_raw_msg.py (hunk 1 above)
Comment (duncan9562) [reply DQ_IL2_index_raw_msg.py]: Latest commit should address this.

**Review by david-wirelab — APPROVED** [review]
LGTM

**Pull request merged**